File: Presentation/webApp/src/app/shared/Addresses.ts
import { Injectable } from '@angular/core';

@Injectable()
export class Addresses {
  stsServerAddress: string = 'http://identity.qtlife.com:5000';
  physicanRedirectUrl: string = 'http://app.qtlife.com:4200/center/dashboard';
  centerGroupRedirectUrl: string = 'http://app.qtlife.com:4200/center/dashboard';
  nambayaUserRedirectUrl: string = 'http://app.qtlife.com:4200/user/dashboard';
  cardiologistRedirectUrl: string = 'http://app.qtlife.com:4200/cardiologist/dashboard';
  pharmacistRedirectUrl: string = 'http://app.qtlife.com:4200/pharmacist/dashboard';
  logoutUrl: string = 'http://app.qtlife.com:4200/logout';
  postLogoutRedirectUri: 'http://app.qtlife.com:4200/logout';
}

@Injectable()
export class APIUrls {
  PharmacistServiceBaseUrl: string = 'http://service.qtlife.com:5002';
  CardiologistServiceBaseUrl: string = 'http://service.qtlife.com:5004';
  centralServiceBaseUrl: string = 'http://service.qtlife.com:5005';
  NambayaUserServiceBaseUrl: string = 'http://service.qtlife.com:5007';
  IdentityServerBaseUrl: string = 'http://identity.qtlife.com:5000';
  PatientServiceBaseUrl: string = 'http://service.qtlife.com:5001';
  LogginServiceBaseUrl: string = 'http://service.qtlife.com:5010';
  UserManagementServiceBaseUrl: string = 'http://service.qtlife.com:5003';
}

File: Presentation/webApp/src/environments/Addresses.rs
import { Injectable } from '@angular/core';

@Injectable()
export class Addresses {
  stsServerAddress: string = 'https://identity.qt-life.nambaya-medical.de';
  physicanRedirectUrl: string = 'https://qt-life.nambaya-medical.de/center/dashboard';
  centerGroupRedirectUrl: string = 'https://qt-life.nambaya-medical.de/center/dashboard';
  nambayaUserRedirectUrl: string = 'https://qt-life.nambaya-medical.de/user/dashboard';
  cardiologistRedirectUrl: string = 'https://qt-life.nambaya-medical.de/cardiologist/dashboard';
  pharmacistRedirectUrl: string = 'https://qt-life.nambaya-medical.de/pharmacist/dashboard';
  logoutUrl: string = 'https://qt-life.nambaya-medical.de/logout';
  postLogoutRedirectUri: 'https://qt-life.nambaya-medical.de/logout';
}

@Injectable()
export class APIUrls {
  PharmacistServiceBaseUrl: string = 'https://services.qt-life.nambaya-medical.de/pharmacist';
  CardiologistServiceBaseUrl: string = 'https://services.qt-life.nambaya-medical.de/cardiologist';
  centralServiceBaseUrl: string = 'https://services.qt-life.nambaya-medical.de/centralgroup';
  NambayaUserServiceBaseUrl: string = 'https://services.qt-life.nambaya-medical.de/nambayauser';
  IdentityServerBaseUrl: string = 'https://identity.qt-life.nambaya-medical.de';
  PatientServiceBaseUrl: string = 'https://services.qt-life.nambaya-medical.de/patient';
  LogginServiceBaseUrl : string = 'https://services.qt-life.nambaya-medical.de/logging';
  UserManagementServiceBaseUrl: string = 'https://services.qt-life.nambaya-medical.de/usermanagement';
}
